### DISCUSSION TITLE
Is there a docker image that include both mysql and python?

### DISCUSSION THREAD

**Discussion by fscarmen** [tiangolo/uvicorn-gunicorn-docker#234] [Questions] [answered]
I hope to find a suitable image here.

**Comment by codespearhead** [comment, marked as answer]
I wonder what use case would justify that combination, because it'd tightly couple the backend and the persistence layer.

In other words, any change to the backend would directly impact the database instance that already has production data.

Even in the pre-containerization era, the database was running elsewhere for this very reason.

Either way, you can always create your own custom Dockerfile to cater to this combination, since no such image appears to exist on DockerHub.
